client/src/components/FriendsBar/FriendSearchForm.tsx

```diff
@@ -0,0 +1,116 @@
+import { useState } from 'react';
+import { Friend } from 'GlobalType';
+import styled from 'styled-components';
+import { HOST } from '../../constants';
+import FriendSearchInput from './FriendSearchInput';
+
+interface FriendSearchResultProps {
+  idx: number;
+  user_id: string;
+  username: string;
+  profileImg: string;
+}
+
+const FriendSearchForm = () => {
+  const [friendObject, setFriendObject] = useState<Friend[] | null>(null);
+  return (
+    <>
+      <FriendSearchContainer>
+        <FriendSearchTitle>새로운 친구 추가</FriendSearchTitle>
+        <FriendSearchInput setFriendObject={setFriendObject} />
+        <FriendResultList>
+          {friendObject &&
+            friendObject.map(({ idx, user_id, username, profile_img }) => {
+              console.log(HOST, profile_img);
+              return <FriendSearchResult idx={idx} user_id={user_id} username={username} profileImg={profile_img} />;
+            })}
+        </FriendResultList>
+        <FriendRequestButton>Friend Request!</FriendRequestButton>
+      </FriendSearchContainer>
+    </>
+  );
+};
+
+export default FriendSearchForm;
+
+const FriendSearchResult = ({ idx, user_id, username, profileImg }: FriendSearchResultProps) => {
+  return (
+    <FriendResult>
+      <FriendProfile imgURL={HOST + '/' + profileImg}></FriendProfile>
```

Comment:
템플릿 리터럴로 변경하면 더 읽기 좋을 것 같습니다
```suggestion
      <FriendProfile imgURL=`${HOST}/${profileImg}` />
```